sift/packages/libbz2-dev.sls

```diff
@@ -0,0 +1,2 @@
+libbz2-dev:
```

Comment:
Need to move these to sift-package-libbz2-dev and set the pkg.installed and `name:` variable.